packages/block-library/src/template-part/edit/index.js

```diff
@@ -53,15 +57,12 @@ export default function TemplatePartEdit( {
 		isMissing,
 		defaultWrapper,
 		area,
-		enableSelection,
```

Comment:
I think we still need this logic(or refactored of course) combined with the check about available patterns. If you have nothing from either it should start blank right away and open the modal for the TP title.

Reply:
cc @jorgefilipecosta we discussed this a bit. I think the current logic was a bit all over the place but yes we still need to handle this somehow.